### DISCUSSION TITLE
Can't prevent dark reader from changing the style of some specific elements.

### DISCUSSION THREAD

**Discussion by ShahriarKh** [darkreader/darkreader#7440] [Q&A]
Hey there!
I'm using dark reader on my website and the package is really great, Thank you for developing it! However, I'm really getting mad because of some CSS issues. some parts of the page don't look right in dark mode.

For example, I have a yellow button with black text. When I turn the dark mode on, the background becomes a little darker and makes text white which is unreadable.

![image](https://user-images.githubusercontent.com/31452340/142767158-0e9bff15-698a-4e89-8aeb-ca3eaed5fe43.png)
![image](https://user-images.githubusercontent.com/31452340/142767137-e84de763-12e6-4442-ad6f-4c5f871da5af.png)
**Expected result:**
(The colors are the same as the first picture)
![image](https://user-images.githubusercontent.com/31452340/142767191-698d0cbd-9a7e-4b5f-9f4f-3a748b148867.png)

----
 Another example is a purple button that turns to blue when dark mode is on!
 
![image](https://user-images.githubusercontent.com/31452340/142767348-3c73e3ac-61cd-4851-91fd-cf52d3d7e4f0.png)
![image](https://user-images.githubusercontent.com/31452340/142767359-d1819fa4-4235-400e-95af-3511b7f0e3d2.png)
**Expected result:**
(The colors are the same as the first picture)
![image](https://user-images.githubusercontent.com/31452340/142767375-cdfec260-32b2-4233-9447-6899f74814e5.png)

----
All I want to do is to prevent dark reader from applying styles to some specific elements. 
I already saw https://github.com/darkreader/darkreader/issues/4056 & https://github.com/darkreader/darkreader/discussions/7318,  but can't get it to work. 
- I tried `ignoreInlineStyle`, but all I see it does is to remove inline attributes; the styles don't get removed. I also tried different selectors: `.class`, `[attribute=value]`, `element`, `element.class`, ... but none of them work.
- I can't use `css`, because I have a lot of elements that should be excluded. It takes a lot of time and effort to write CSS for every excluded item
- I tried 'darkreader' class, but it doesn't work. It also has a weird behavior: the element is removed from the DOM when I turn off the dark mode and never comes back :-)

If it helps, I'm using `next.js 11.1.2` + `scss (sass 1.43.4)` and `darkreader 4.9.4`
# What should I do?

**Comment by Gusted** [comment]
> What should I do?

Use the `css` option, it's the only way to get such results, there isn't a magic button that fixes these issues.

**Reply by ShahriarKh** [reply]
Are we going to have such an option in v5? Redeclaring all of those classes that should be ignored is overkill! I have like 216 semantic color combinations...

**Reply by Gusted** [reply]
> Are we going to have such an option in v5?

It's already a option:
https://github.com/darkreader/darkreader/blob/master/index.d.ts#L142

**Reply by ShahriarKh** [reply]
I meant the "magic button" option 🥲

**Reply by Gusted** [reply]
I wish there was one. But it's pure magic.

**Comment by MeguminChan830** [comment]
I think you could use element.style.setProperty("color", "green", "important") as a temporary workaround.
